diff --git a/gcc/gcc.cc b/gcc/gcc.cc
index 506c2acc282..e4ad71d18c7
--- a/gcc/gcc.cc
+++ b/gcc/gcc.cc
@@ -1335,9 +1335,7 @@ static const char *const multilib_defaults_raw[] = MULTILIB_DEFAULTS;
 /* Linking to libgomp implies pthreads.  This is particularly important
    for targets that use different start files and suchlike.  */
 #ifndef GOMP_SELF_SPECS
-#define GOMP_SELF_SPECS \
-  "%{fopenacc|fopenmp|%:gt(%{ftree-parallelize-loops=*:%*} 1): " \
-  "-pthread}"
+#define GOMP_SELF_SPECS ""
 #endif
 
 /* Likewise for -fgnu-tm.  */
